samples/besu/simpleasset/contracts/transferInterface.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
This also seems to be a core file, (i.e. not application dependent), so it should also go in `core/networks/besu` folder.
May be you can create one folder there `interfaces/asset-exchange` and put it there. So it would be like `core/networks/besu/interfaces/asset-exchange/transferInterface.sol`